build-tools/installers/create-installers.targets

```diff
@@ -165,9 +195,12 @@
     <_MSBuildFiles Include="$(MSBuildSrcDir)\Xamarin.Android.Analysis.targets" />
     <_MSBuildFiles Include="$(MSBuildSrcDir)\Xamarin.Android.Bindings.targets" />
     <_MSBuildFiles Include="$(MSBuildSrcDir)\Xamarin.Android.Build.Tasks.dll" />
+    <_MSBuildFiles Include="$(MSBuildSrcDir)\Xamarin.Android.Build.Tasks.pdb" />
```

Comment:
I think we need to update `Xamarin.Android.Build.Tasks.csproj` so that the Release configuration actually emits `.pdb` files:

```xml
    <DebugType>portable</DebugType>
    <DebugSymbols>True</DebugSymbols>
```